VMWare/vmware-vcenter/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,108 @@
+[
+  {
+    "value": "The event is generate by {user.name} with the ip address: {vmware_vcenter.ip_address}",
+    "conditions": [
+      {
+        "field": "vmware_vcenter.ip_address"
+      },
+      {
+        "field": "user.name"
+      }
+    ]
+  },{
+    "value": "The event is generate by the user agent {user_agent.original} with the ip address: {vmware_vcenter.ip_address} at the login time {vmware_vcenter.login_time}",
+    "conditions": [
+      {
+        "field": "vmware_vcenter.ip_address"
+      },
+      {
+        "field": "user_agent.original"
+      },
+      {
+        "field": "vmware_vcenter.login_time"
+      }
+    ]
+  },
```

Comment:
I think that these smart-description match session login. It should be better to have `User {user.name} log in {host.ip}` or  `User {user.name} log out from {host.ip}`